src/libs/export.c

```diff
@@ -122,6 +132,81 @@ static void _mouse_over_image_callback(gpointer instance, dt_lib_module_t *self)
   dt_lib_queue_postponed_update(self, _update);
 }
 
+bool _is_int(double value)
+{
+    return (value == (int)value);
+}
+
+static void _scale_optim()
+{
+  double a=1, b=1;
+  dt_imageio_scale_splitup(&a, &b);
```

Comment:
So my proposal is more something like:

```
gchar *scale_str = dt_conf_get_string(CONFIG_PREFIX "resizing_factor");
dt_imageio_scale_splitup(scale_str, &a, &b);
```

And a, b could probably be renamed num, denum (for numerator denominator).